nekoyume/Assets/_Scripts/BlockChain/ActionManager.cs

```diff
@@ -440,11 +458,7 @@ List<Guid> consumableIds
                 .First()
                 .ObserveOnMainThread()
                 .Timeout(ActionTimeout)
-                .DoOnError(e =>
-                {
-                    HandleException(action.Id, e);
-                    ActionRenderHandler.BackToMain(false, e);
-                });
+                .DoOnError(e => HandleException(action.Id, e));
```

Comment:
I want know why I didn't use `try catch` only for this part

Reply:
I'll fix this.